Sources/BeaconChain/DataStructures/State/Eth1Data.swift

```diff
@@ -1,6 +1,10 @@
 import Foundation
 
-struct Eth1Data {
+struct Eth1Data: Equatable {
     let depositRoot: Data
     let blockHash: Data
+
+    public static func == (lhs: Eth1Data, rhs: Eth1Data) -> Bool {
```

Comment:
No need to add this, Swift 4 will synthesize.

Reply:
I'm using swift 4.2 and it definitely seems to require it.